### PR TITLE
Visual fixes and improvements

### PR DESCRIPTION
**Pull request by Malex14**
This PR fixes a lot of small CSS-Issues and improves the cpu and memory charts.

## Charts
I've updated the chart library to the latest version and added a tooltip that displays the cpu / ram usage at the point of time where the cursor hovers over the chart. Additionally the cpu chart and legend show the usage now in percent (instead of 0 to cpu count) and has its y max fixed at 100%. I've also added a hover text to the cpu legend that displays the averages of the past 1, 5 and 15 minutes.  Also the y-Axis is now readable in dark mode:
<details>
<summary>Click to show screenshots</summary>

### Improved:

</details>

## Overflowing Boxes

To fix the mobile overflows, I mostly just reenabled the commented out `@media` rules. But a few other changes had to be made too:
<details>
<summary>Click to show screenshots</summary>

### Improved:

</details>

## Overlapping Text

Lastly there was some overlapping text in the "External monitoring tool" section:
<details>
<summary>Click to show screenshots</summary>

### Improved:

</details>

### CODE REVIEW

**Comment by Malex14** [comment]
I just rebased the pr on top of the current master branch

**Comment by Malex14** [comment]
Hello @kesselb, since you reviewed other PRs in the past, can you (or maybe someone else) please take a look at this PR?

**Comment by kesselb** [comment]
Hi, 

Thank you so much for your pull request—it’s much appreciated!

Could you please update the implementation of Linux.getCpuCount to use /proc/cpuinfo, similar to how we handle it in Linux.getCpuName? I’d like to avoid executing shell commands where possible to keep things consistent and efficient.

Let me know if you have any questions or need help with this!

**Comment by Malex14** [comment]
Thank you for the quick response. I've changed the function as you asked.

**Comment by joshtrichards** [comment]
Linked two Issues this likely closes.

File: lib/OperatingSystems/FreeBSD.php [hunk 1]
```diff
@@ -75,6 +75,17 @@ public function getCpuName(): string {
 		return $data;
 	}
 
+	public function getCpuCount(): int {
+		$numCpu = -1;
+
+		try {
+			$numCpu = intval($this->executeCommand('sysctl -n hw.ncpu')); //TODO: this should be tested if it actually works on FreeBSD
```
Comment (kesselb) [inline lib/OperatingSystems/FreeBSD.php:82]: That yields the expected result for my cpu.

**Review by kesselb — APPROVED** [review]

File: lib/OperatingSystems/FreeBSD.php (hunk 1 above)
Comment (Malex14) [reply lib/OperatingSystems/FreeBSD.php:82]: ok :+1: , then I will remove the comment

**Comment by Malex14** [comment]
should I rebase the commits on top of master again?

**Pull request merged**